components/landing/TheQuickStart/PrerequisitesForWindows.vue

```diff
@@ -1,22 +1,9 @@
 <template>
   <div class="prerequisites">
     <div class="prerequisites__section">
-      <p>
-        On Windows you need to use Anaconda3 or Miniconda3 to install all the
-        dependencies.
-      </p>
-    </div>
-    <div class="prerequisites__section">
-      <h3 class="h4">
-        Install the compiler
-      </h3>
-      <pre class="code prerequisites__code">conda install --update-deps vs2017_win-64 vs2017_win-32 msvc_runtime</pre>
-    </div>
-    <div class="prerequisites__section">
-      <h3 class="h4">
-        Install OpenBLAS development headers
-      </h3>
-      <pre class="code prerequisites__code">conda install --update-deps -c conda-forge -y openblas cmake</pre>
+      <!-- eslint-disable vue/singleline-html-element-content-newline -->
+      <p><strong>Windows 64-bit:</strong> Download and run the <AppLink url="https://win.rustup.rs/x86_64">64-bit installer</AppLink>, which will provider follow up instructions.</p>
```

Comment:
Same: it would be nice to add segment tracking

Reply:
Added segment info here https://github.com/Qiskit/qiskit.org/pull/2904/commits/e4f624245eefd7dc5dc2fbe21522256669d6e091